base/collection.go

```diff
@@ -110,7 +110,11 @@ func GetCollectionFromCluster(cluster *gocb.Cluster, spec BucketSpec, waitUntilR
 	}
 
 	// Set limits for concurrent query and kv ops
-	collection.queryOps = make(chan struct{}, MaxConcurrentQueryOps)
+	maxConcurrentQueryOps := MaxConcurrentQueryOps
```

Comment:
Maybe rename the const while were here, so it's clear how it's used.

```suggestion
	maxConcurrentQueryOps := DefaultMaxConcurrentQueryOps
```